### PR TITLE
Add main page heading for users with power of attorney

### PR DESCRIPTION
**Pull request by sonwit**
## Related Issue(s)
- #946 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/rtk/features/userInfo/userInfoApi.ts
Comment (mgunnerud) [inline src/rtk/features/userInfo/userInfoApi.ts:24]: Could we replace this `any` with the returned type? Looks like both of these requests return type `ReporteeResponse`.

File: src/rtk/features/userInfo/userInfoApi.ts
Comment (mgunnerud) [inline src/rtk/features/userInfo/userInfoApi.ts:31]: Also this any

File: src/components/UserInfoBar/UserInfoBar.tsx
Comment (mgunnerud) [inline src/components/UserInfoBar/UserInfoBar.tsx:21]: Looks like we might be missing a condition here? Previously the code checked for `userInfoName !== reporteeName && reporteeName`, while now it checks that the values are not equal. (not sure if it will ever happend that an empty `reporteeData.name` is shown)

**Review by mgunnerud — CHANGES_REQUESTED** [review]
Looks great! 😄added some minor comments

File: src/components/UserInfoBar/UserInfoBar.tsx
Comment (allinox) [inline src/components/UserInfoBar/UserInfoBar.tsx:22]: Good catch with the p and the span! 😊

File: src/rtk/features/userInfo/userInfoApi.ts
Comment (allinox) [reply src/rtk/features/userInfo/userInfoApi.ts:24]: I agree 🙌

**Review by allinox — COMMENTED** [review]
Good job! 😄

I don't really have much to add to Martin's assessment, so I'll just leave it to him to approve this PR when his comments have been addressed.

File: src/rtk/features/userInfo/userInfoApi.ts
Comment (sonwit) [reply src/rtk/features/userInfo/userInfoApi.ts:24]: Absolutely. I was leaning a litte to much on code generated by Copilot here. Specifying return types enhances clarity 👍.

File: src/components/UserInfoBar/UserInfoBar.tsx
Comment (sonwit) [reply src/components/UserInfoBar/UserInfoBar.tsx:22]: semantics  ♡

File: src/components/UserInfoBar/UserInfoBar.tsx
Comment (sonwit) [reply src/components/UserInfoBar/UserInfoBar.tsx]: You're absolutely right. We should only display "for [reporteeData.name]" if reporteeData.name exists and is different from user.name. I'll update the condition to ensure this. Thanks for catching it!

**Review by mgunnerud — APPROVED** [review]
LGTM 😄

**Pull request merged**
